test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -41,7 +41,7 @@ function ExecuteRequestWithOutFile
         [string]
         $cmdletName,
         [string]
-        $uri = "http://httpbin.org/get"
+        $uri = (Get-WebListenerUrl -Test 'Get')
```

Comment:
Please remove parentheses.
We use this many times - can we move the line to BeforeAll?

It seems the time has come to add `ValidateSet()` to `Get-WebListenerUrl` parameters.

Reply:
Unless I'm mistaken, the Parens are required when assign a default value with a statement in a Param block. At your suggestion I tried removing them but I get all kinds of errors.

As for moving this to the BeforeAll block, I don't think that makes sense in this context since this is part of the `ExecuteRequestWithOutFile` function. 

For the more general concern, we could create a variable, but, as we move the other tests over you will see that it will get less possible to do so in the `BeforeAll` Block without losing meaning in the `It` blocks (in the case where there will be a query string required as well). I would prefer we keep the URL generation to each `It` block for consistency.

Reply:
Ah, sorry - My thoughts was about many `Get-WebListenerUrl -Test 'Get'` in the file - I see only 10 times so I think we can keep it as is.